cobalt/loader/resource_cache.cc

```diff
@@ -289,8 +289,9 @@ void ResourceCacheBase::NotifyResourceLoadingRetryScheduled(
 
 void ResourceCacheBase::ReclaimMemoryAndMaybeProcessPendingCallbacks(
     uint32 bytes_to_reclaim_down_to) {
+  bool on_conceal = false;
```

Comment:
Tiny nit: this reads a little odd here, but maybe better than commenting on ,`false` /* on_conceal /*` in the argument passing. Maybe move `log_warning_if_over` also into const bool then ?

Reply:
I just didn't do this because log_warning_if_over changes values below